### ISSUE TITLE
agora abrir o help explode o pd

### ISSUE THREAD

**Issue by porres** [charlesneimog/py4pd#69]
instalou, mas te isso... 

agora, faz favir de aumentar um pouco a fonte e nao deixar a janela do patch tao grande além do que tem de código

**Comment by charlesneimog** [comment]
É que eu uso o zoom no Pd, até tava pensando em como fazer isso para outros usuarios, no linux o Pd tem uma fonte muito pequena. rsrs... Fazer isso!

Vou ver oq ta rolando.

**Comment by porres** [comment]
vai subir uma nova versao?

queria testar as coisas de partitura

to numas pira de trabalhar com um gerador de acorde e ja tenho um gerador de escala na ELSE, bora fazer esse bach aí

**Comment by charlesneimog** [comment]
Eu tô resolvendo uns roles para o Linux, daí vou subir uma nova versão...

**Comment by charlesneimog** [comment]
Simm, acho 1ue vai ser legal...

**Comment by porres** [comment]
como é, é um objeto único essa parada?

**Comment by charlesneimog** [comment]
Por enquando eu so fiz um, que chama `py.chord`. Mas minha ideia a longo prazo é fazer um `py.voice`, `py.note`. Igual o OpenMusic.

Mas já vi que preciso dar uma olhada no código para Mac ainda, to usando um código interno que le png, mas parece nao funciona em Mac ainda.

**Comment by porres** [comment]
mas o external do pd é um objetozinho só mesmo né

**Comment by charlesneimog** [comment]
É isso aqui:

https://github.com/charlesneimog/pd-score

Ele até funciona no mac, mas ainda não consegue gerar as notas igual no linux.

**Comment by charlesneimog** [comment]
Mas vale ressaltar, que isso, uma vez que o py4pd funcione direito, é clicar e instalar. E mais de boas, não tem problema entre plataformas, etc.

**Comment by porres** [comment]
> mas o external do pd é um objetozinho só mesmo né

?

**Comment by charlesneimog** [comment]
não, vão ser scripts de Pythons, quem 'gera' o objeto para Pd é o py4pd. Sacou?

É que eu n sei se vc esta perguntando do binarios ou nao. Se for dos binarios, não vão ter binários não. O único binário é o py4pd.

**Comment by porres** [comment]
Tendi, tipo pdlua

**Comment by charlesneimog** [comment]
ahhh, exatamente!

**Comment by porres** [comment]
> É isso aqui:
> 
> https://github.com/charlesneimog/pd-score

lindo, quero! Arruma esse objeto e atualiza pra eu brincar!

**Comment by porres** [comment]
Tenho um bom nome prima lib de CAC

**Comment by charlesneimog** [comment]
@porres eu atualizei o `py4pd` para a versão 0.8.3 e subi o `pd-score` 0.0.0-beta no deken rsrs. Devia ter chamado de `alpha`.

Na minha máquina virtual com o Mac funciona, me fala ai se funciona.

**Comment by porres** [comment]
![Screen Shot 2023-12-02 at 01 56 28](https://github.com/charlesneimog/py4pd/assets/5360142/d9af5429-abea-415e-9018-c2ac2e5e31d1)

**Comment by charlesneimog** [comment]
E isso mesmo, depois q vc instalou a neoscore funcionou?

**Comment by porres** [comment]
> A pasta pd-score ta nos PATHS do Pd?
sim

> depois q vc instalou a neoscore funcionou?
instalei o q? srsrsrsrs

como faz isso?

**Comment by porres** [comment]
tentando aqui, eu nao li nada, vc usa uma fonte too fucking small

**Comment by porres** [comment]
instalei e deu mema merda

![Screen Shot 2023-12-02 at 11 36 27](https://github.com/charlesneimog/py4pd/assets/5360142/f701f2ec-19dc-47c2-9409-43743c42c700)

**Comment by porres** [comment]
ah, tem q instalar numpy agora pelo jeito

**Comment by porres** [comment]
tem como ajustar o tamanho dele nao? e quando eu tenho um dó e um ré, um sobrepõe no outro, tem como jogar pro lado? (open music deve fazer isso né)

**Comment by charlesneimog** [comment]
Faz, mas isso tem que fazer o código do `pd-score` ainda. Se você ver la, esse código tem um 20 linhas só, depois tem que adicionar essas outras regras de partitura. Mas primeiro eu quero fazer o `py4pd` funcionar direito.

Depois também tem que implementar um jeito para redimensionar, mas daí tem que pensar como fazer isso.

**Comment by charlesneimog** [comment]
Mas se funciona já é um grande passso HAHAHAH.
